src/App.jsx

```diff
@@ -65,100 +58,127 @@ const styles = theme => ({
     padding: theme.spacing.unit * 3,
     minWidth: 0, // So the Typography noWrap works
   },
+  formControlLabel: {
+    display: 'flex',
+    align: 'center',
+    marginLeft: 0,
+  },
   toolbar: theme.mixins.toolbar,
 });
-
 const bugType = ['Unassigned Bug', 'Assigned Bugs', 'Simple Bugs'];
 
 class App extends Component {
-
   constructor(props) {
     super(props);
-    var allProjects = [...data.Projs];
-    allProjects.forEach(function(project, index) {
-      project.push(false)
-    })
+
+    const allProjectGroups = projectInfo;
+    // Contains all information from allProjectGroups parsed
+    const allProjectGroupsEntries = [];
+    // Contains a key-value pair as [projectkey]:[checked]
+    const projectSelected = {};
+
+    // Organizes the information so that it can be displayed easily
+    Object.entries(allProjectGroups).forEach(element => {
+      const curr = {};
+      const [key, value] = element;
+
+      curr.group = key;
+      curr.projects = Object.entries(value).reduce((projects, project) => {
+        const [key, proj] = project;
+
+        proj.key = key;
+        projects.push(proj);
+        projectSelected[key] = false;
+
+        return projects;
+      }, []);
+      allProjectGroupsEntries.push(curr);
+    });
```

Comment:
If we have complete control of the yaml files, could we organize them in a way that wouldn't require organizing them after loading them?

Reply:
Yeah I really hope it could be that way too. The problem I face so far is when it is recognized as key value pair and I need both of them.. 

`Name: Accessibility
InfoHeader: Extra information for Accessibility
SubInfo: Get involved with the [Accessibility](https://wiki.mozilla.org/Accessibility/Contribute) team
Products: 
 - Core: ['Disability Access APIs']
 - Firefox: ['Disability Access']
 - additional: {'keywords': ['access']}`

in the above, previously, we load the file and import it using the file name 'a11y' as the key and the above as the value. But then I realized it might be better to keep the 'a11y' as well, to later be the value of the checkbox and use 'Accessibility' as the displayed name only as 'Accessibility' might have duplicate in other SuperGroup  or i might need 'a11y' as the query later..

I initially think of adding 'Key: a11y' but i am afraid it will be redundant and I think later we might face similar problem when handling the Products (if we need both the 'Core' and ['Disability Access APIs'] )